test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,37 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception{
+
+        //no null objects
+        assertNotNull();
+        assertNotNull("a");
+        assertNotNull("aa","aa");
+        assertNotNull("a","b","c");
+
+        //only one null
+        assertIsNull((Object)null);
+
+        //some null objects
+        assertIsNull("a",null);
+        assertIsNull("a",1,null);
+
+        //all null objects
+        assertIsNull(null,null);
+        assertIsNull(null,null,null);
+
+
+
+    }
+
+    private void assertIsNull(Object... objects){
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
You may try naming according to the 3-part scheme for test methods which are specific.